java/email-contact-form/src/Main.java

```diff
@@ -0,0 +1,133 @@
+package io.openruntimes.java.src;
+
+import io.openruntimes.java.RuntimeContext;
+import io.openruntimes.java.RuntimeOutput;
+import java.util.Map;
+import java.util.List;
+import java.util.HashMap;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.lang.*;
+import io.appwrite.Client;
+import jakarta.mail.MessagingException;
+
+
+public class Main {
+
+    public RuntimeOutput main(RuntimeContext context) throws Exception {
+
+        List<String> requiredEnvVariables = Arrays.asList(
+                "SUBMIT_EMAIL",
+                "SMTP_HOST",
+                "SMTP_USERNAME",
+                "SMTP_PASSWORD"
+        );
+
+        Utils.throwIfMissing(System.getenv(), requiredEnvVariables);
+        if (System.getenv("ALLOWED_ORIGINS").equals("*")) {
+            context.log("WARNING: Allowing requests from any origin - this is a security risk!");
+        }
+
+        if (context.getReq().getMethod().equals("GET")) {
+            return context.getRes().send(
+                    Utils.getHtmlContent("index.html"),
+                    200,
+                    Map.of("content-type", "text/html")
+            );
+        }
+
+
+        if(!context.getReq().getHeaders().get("content-type").equals("application/x-www-form-urlencoded")){
+            context.error("Incorrect content type");
+            String referer = context.getReq().getHeaders().get("referer");
+            String errorCode = ErrorCode.INVALID_REQUEST;
+            return context.getRes().redirect(
+                    String.format("%s?code=%s", referer, errorCode)
+            );
+        }
+
+        if(!Cors.isOriginPermitted(context)){
+            context.error("Origin not permitted");
+            String referer = context.getReq().getHeaders().get("referer");
+            String errorCode = ErrorCode.INVALID_REQUEST;
```

Comment:
I don't think we need this variable, just move it inline

Reply:
Removed the errorCode variable and moved errorCode it to inline